CHANGELOG/CHANGELOG-1.4.md

```diff
@@ -1,6 +1,32 @@
 
 <hr>
 
+## v1.4.0-alpha.1(TBD)
+
+### BoltDB
```

Comment:
https://github.com/etcd-io/bbolt/compare/v1.4.0-alpha.0...51926613987d57aa9df1f1bfcce0467598932e90

Do we need to mention the change about golang version?

Reply:
updated & thx